ReactiveCocoa/Swift/Property.swift

```diff
@@ -95,6 +95,57 @@ extension MutableProperty: SinkType {
 	}
 }
 
+/// Wraps a `dynamic` property, or one defined in Objective-C, using Key-Value
+/// Coding and Key-Value Observing.
+///
+/// Use this class only as a last resort! `MutableProperty` is generally better
+/// unless KVC/KVO is required by the API you're using (for example,
+/// `NSOperation`).
+@objc public final class DynamicProperty: NSObject, PropertyType {
+	public typealias Value = AnyObject?
```

Comment:
My biggest question is, what should happen if the cast fails? You get `nil`? That might make sense, but could also be surprising.

This keeps it simple and pushes that complexity to the consumer.